trinity/utils/chains.py

```diff
@@ -100,21 +100,20 @@ def load_nodekey(nodekey_path):
 class ChainConfig:
     chain_identifier = None
 
-    _data_dir = None
-    _nodekey_path = None
-    _nodekey = None
-    _network_id = None
+    _data_dir:str = None
+    _nodekey_path:str = None
+    _nodekey:str = None
+    _network_id:int = None
 
     def __init__(self,
-                 chain_identifier,
-                 data_dir=None,
-                 nodekey_path=None,
-                 nodekey=None,
-                 network_id=None):
+                 chain_identifier:str,
+                 data_dir:str=None,
```

Comment:
for consistent formatting, the convention setup thus far in the `p2p` module is for there to be a space after the `:` here.

```
def fn(self,
       chain_identifier: str,
       ...
```

Reply:
No problem. Is there a linter I can run locally?